modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/request/SearchResponseImpl.java

```diff
@@ -75,8 +72,14 @@ public SearchSettings getSearchSettings() {
 		return _searchSettings;
 	}
 
-	public Optional<String> getSpellCheckSuggestionOptional() {
-		return SearchStringUtil.maybe(_hits.getCollatedSpellCheckResult());
+	public String getSpellCheckSuggestion() {
+		String collatedSpellCheckResult = _hits.getCollatedSpellCheckResult();
```

Comment:
@julschong You missed this comment https://github.com/liferay-core-infra/liferay-portal/pull/3438#discussion_r1113885664